tools/doc/json.js

```diff
@@ -243,7 +243,6 @@ function processList(section) {
         current.options.push(n);
         current = n;
       }
-      return;
```

Comment:
This is even a stronger case for readability.
this return is 16 lines above the end of the function...